File: 20221012/모범답안/20221012_02/remodeling.py
def solution(now, tobe):
  '''
  :param now: list
  :param tobe: list
  :return: int
  '''
  answer = 0

  for i, j in zip(now, tobe):
      if i < j:
          answer += j-i
      elif i > j:
          answer += j+i
  
  return answer

File: 20221012/모범답안/20221012_02/remodeling.js
/**
 * @param now {Array}
 * @param tobe {Array}
 */
function solution(now, tobe) {
    var answer = 0
    var c = now.map(function(e, i) {
        return [e, tobe[i]];
    });

    for (var i in c) {
        if (c[i][0] < c[i][1]) {
            answer += c[i][1] - c[i][0]
        } else if (c[i][0] > c[i][1]) {
            answer += c[i][1] + c[i][0]
        }
    }

    return answer
}